tests/components/mqtt/test_update.py

```diff
@@ -527,3 +528,12 @@ async def test_unload_entry(hass, mqtt_mock_entry_with_yaml_config, tmp_path):
     await help_test_unload_config_entry_with_platform(
         hass, mqtt_mock_entry_with_yaml_config, tmp_path, domain, config
     )
+
+
+async def test_reloadable(hass, mqtt_mock_entry_with_yaml_config, caplog, tmp_path):
+    """Test reloading the MQTT platform."""
+    domain = update.DOMAIN
+    config = DEFAULT_CONFIG
+    await help_test_reloadable(
+        hass, mqtt_mock_entry_with_yaml_config, caplog, tmp_path, domain, config
+    )
```

Comment:
Is this for another PR?

Reply:
It is not, but the test did not work before because it also depended on the legacy setup. That is fixed now. To be compliant with the other platform tests and to make sure the common test is okay with the update platform, this test is added now.